contrib/search/confluence/README.md

```diff
@@ -0,0 +1,6 @@
+# Confluence
+
+These files help you add Confluence as a source to the Backstage Search plugin.
+To do so, add both files in this directory under the packages/backend/src/plugins/search/ pathway in your Backstage app as TypeScript files.
+Then, update your packages/app/src/components/search/SearchPage.tsx and packages/backend/src/plugins/search.ts
+to include the new Search source.
```

Comment:
Do you want to add two examples here of how to include it. 

For example in `packages/backend/src/plugins/search.ts` something like:

```ts
indexBuilder.addCollator({
  defaultRefreshIntervalSeconds: 600,
  collator: // instantiate ConfluenceCollator
});
```

and for `packages/app/src/components/search/SearchPage.tsx` something like:

```tsx
case 'confluence':
            return (
              < ConfluenceResultListItem
                key={result.document.location}
                result={result.document}
              />
            );
          // ...
```

Reply:
Good call, added code snippets to the README!